docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.8'
+
+# Ports
+# Builder: 3000
+# Explorer: 3001
+# Manager: 3002
+# Pinata: 3003
+# Indexer: 3004
+# Dev chain 1: 3005
+# Subgraph dev chain 1: 3006
+# Dev chain 2: 3007
+# Subgraph dev chain 2: 3008
+
+services:
+  pinata:
+    image: ghcr.io/gitcoinco/pina:init
+    ports:
+      - '127.0.0.1:3003:8000'
+
+  localchain-1:
+    image: ghcr.io/gitcoinco/allo-contracts-v1:pr-1
+    ports:
+      - '127.0.0.1:3005:8545'
+    environment:
+      - DEV_CHAIN_ID=313371
+
+  localchain-1-init:
+    image: ghcr.io/gitcoinco/allo-contracts-v1:pr-1
+    depends_on:
+      - pinata
+      - localchain-1
+    environment:
+      - PINATA_HOST=pinata
```

Comment:
Also if this is initializing not just a chain but also IPFS, maybe there's a better name that `localCHAIN-init`.

Could it make sense to have just one init container that depends on `pinata`, `localchain-1`, and `localchain-2`, and call it `init` or something like that?